swift_browser_ui_frontend/src/views/Containers.vue

```diff
@@ -303,8 +360,17 @@ export default {
     active() {
       return this.$store.state.active;
     },
-    openCreateFolderModal() {
-      return this.$store.state.openCreateFolderModal;
+    openCreateFolderModal: {
+      get() {
+        return this.$store.state.openCreateFolderModal;
+      },
+      set() {},
```

Comment:
Is empty setter needed? I don't mind it being there but just wondering if it has a meaning.

Reply:
I had to add **get** and **set** there because without them, I had the warnings from console that those methods were lacking. I haven't tried other solutions or looked further for any deeper reasons. Probably you could comment them out and see.

Reply:
For some reason `openCreateFolderModal` property works fine without setter but `openUploadModal` gives warning. I have no better solution for this so I'm happy with this approach.

Reply:
I figured one of the reasons is that if we use a computed property on `v-model` it needs to have a **setter**.